src/server/controllers/VerificationController.ts

```diff
@@ -619,16 +619,48 @@ export default class VerificationController
       );
 
     this.router.route(["/verify/etherscan"]).post(
-      // TODO: add validation
+      body("address")
+        .exists()
+        .bail()
+        .custom(
+          (address, { req }) => (req.addresses = validateAddresses(address))
+        ),
+      body("chainId")
+        .optional()
+        .custom(
+          (chainId, { req }) =>
+            // Support both `body.chain` and `body.chainId`
+            (req.body.chain = chainId)
+        ),
+      body("chain")
+        .exists()
+        .bail()
+        .custom((chain, { req }) => (req.chain = checkChainId(chain))),
```

Comment:
`req.chain` or `req.body.chain`?

Reply:
It's already at `req.body.chain` but should've not assigned to `req.chain` like in https://github.com/ethereum/sourcify/blob/d8e7ef2c7735a9d4cee1121243e17b01e2e24979/src/server/controllers/VerificationController.ts#L663